playbooks/provision.yml

```diff
@@ -79,6 +86,12 @@
             capi_cluster_kubeconfig_path: "{{ ansible_env.HOME }}/kubeconfig-{{ capi_cluster_release_name }}.yaml"
       when: install_mode == 'ha'
 
+    # Since capi_cluster role already installs vanilla kube-prometheus-stack, we can include this role AFTER the
+    # capi_cluster role so that HA deployments will simply receive an updated chart with additional chart values
+    # which include monitoring and alerting config and so that singlenode/demo deployments will also have the 
+    # same monitoring stack installed from scratch
+    - include_role:
+        name: stackhpc.azimuth_ops.kube_prometheus_stack
```

Comment:
This will result in the `HelmRelease` objects becoming out-of-sync with the Helm release, which may cause issues later. In particular, the changes made here are likely to be overridden by the addon operator at some point.